main.html

```diff
@@ -1,47 +1,72 @@
+                <img src="lyft_logo.png" class ="header__image-source" alt="lyft-logo"/>
             </figure>
             <div class="header__small-container">
                 <div class="header__search">
-                    <textarea rows="3" cols="1" placeholder="Where are you go" class="header__textarea"></textarea>
+                    <textarea rows="3" cols="1" placeholder="Where are you going?" class="header__textarea"></textarea>
                     <div class="header__glass">
                         <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" class="header__glass-source" viewBox="0 0 16 16">
                             <path d="M11.742 10.344a6.5 6.5 0 1 0-1.397 1.398h-.001c.03.04.062.078.098.115l3.85 3.85a1 1 0 0 0 1.415-1.414l-3.85-3.85a1.007 1.007 0 0 0-.115-.1zM12 6.5a5.5 5.5 0 1 1-11 0 5.5 5.5 0 0 1 11 0z"/>
-                          </svg>
+                        </svg>
+                    </div>
+                    <div class="header__point">
+                        <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" class="header__point-img" viewBox="0 0 16 16">
+                            <path d="M16 8A8 8 0 1 1 0 8a8 8 0 0 1 16 0zm-8 3a3 3 0 1 0 0-6 3 3 0 0 0 0 6z"/>
+                        </svg>
                     </div>
                 </div>
                 <div class="header__burger">
-                    <img src="burger.png" alt=""/>
+                    <img src="burger.png" class="header__burger-img" alt="burger-button"/>
                 </div>
+                <nav class="header__nav">
+                    <div class="header__list">
+                        <span class="header__text">driver</span>
+                    </div>
+                    <div class="header__list">
+                        <span class="header__text">rider</span>
+                    </div>
+                    <div class="header__list">
+                        <span class="header__text">business</span>
+                    </div>
+                    <div class="header__list">
+                        <span class="header__text">log in</span>
+                    </div>
+                    <div class="header__list-sign">
+                        <span class="header__text header__text--long">sign up</span>
+                    </div>
+                </nav>
             </div>
         </div>
     </header>
-    <main>
+    <div>
         <div class="let-ride">
-            <h1 class="let-ride__header">Let's ride</h1>
-            <div class="let-ride__buttons">
-                <button class="let-ride__apply-button"> Apply to drive
-                </button>
-                <button class="let-ride__sign-button"> Sign up to ride 
-                </button>
-        </div>
-        <img class="let-ride__image" src="main__image.webp" alt=""/>
+            <div class="let-ride__container">
+                <div class="let-ride__container_small">
```

Comment:
can we remove this tag? and use margin: 0 auto for children? always try to avoid redundant nesting

Reply:
I used it here only for flexbox reasons. Tried to remove it. But looks like I'll need these elements for a good desktop layout./